src/style.css

```diff
@@ -0,0 +1,135 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.productCard {
+  width: 198px;
```

Comment:
Card width should be equal 200px (198px + 2px for borders)
You can add `box-sizing: border-box` property, and 2px of borders will be calculated in 200px of full card